docs/_docs/templates.md

```diff
@@ -558,19 +686,27 @@ You can also use this tag to create a link in Markdown as follows:
 {% endraw %}
 ```
 
-### Post URL
+Including {% raw %}{{site.baseurl}}{% endraw %} is optional -- it depends on whether you want the link to be absolute or root-relative.
```

Comment:
~~Instead of all of this, can we just show the new URL filter in action?~~

Nope, because filters don't work with tags. _sigh_